### DISCUSSION TITLE
Migrate longhorn installation from kubectl to helm

### DISCUSSION THREAD

**Discussion by unicode-it** [longhorn/longhorn#9234] [General]
How is it possible to migrate a longhorn installation without data loss from a kubectl deployment to a helm deployment?

Kubectl deployment misses network policies, which the helm chart has. The easiest way to get network policies seems to be to deploy with helm. Does anybody maybe has any ideas about that?

**Comment by robotarmy** [comment]
@unicode-it - I am also interested in this. I recently upgraded to 1.8.0 via Kubectl apply but i would like to move to helm managed
